### PR TITLE
Portfolio mobile UI and layout 

### PR DESCRIPTION
**Pull request by Obimbo07**
This pull request introduces:

1. The nav and banner section UI
2. mobile layout and design

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -13,8 +13,8 @@ nav {
   position: absolute;
   width: 100%;
   height: 72px;
-  left: 0px;
-  top: 0px;
+  left: 0;
```
Comment (brainconnect93) [inline styles.css]: - [ ] Hello @Obimbo07 , Great effort working on your portfolio project (mobile first). But i currently notice the project is not aligning properly to the mobile screen size of `375px` as requested in the project requirement. I would kindly suggest you work on the mobile view of your project as much as possible following the Figma design template given strictly.
Below is the screen recording of how your project looks like in mobile view 👇 

[screen-capture.webm](https://user-images.githubusercontent.com/78634841/235486820-f07beb2c-1dd3-4f09-8a6a-2bceca92118e.webm)

**Review by brainconnect93 — CHANGES_REQUESTED** [review]
### Hi @Obimbo07, 👋 

# STATUS: Changes Required :recycle:

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### To Highlight :1st_place_medal: 

- Descriptive Pull Request title and summary. ✔️  
- Portfolio: setup and mobile first ✔️  
- Follow the Figma Design Template ✔️  
- Touchable mobile screen size of `375px` ✔️  
- Good use of Github flow ✔️ 

### Changes Required

- [ ] I could not highlight this issue using an inline comment because your `HTML` file is not part of your files changes. 
- I would kindly suggest you have only your hamburger icon inside of the `nav` tag.
As stated in the requirement, The hamburger button is on the right, and has a nav tag.
     - The nav tag is for semantics reason, and the right position for convention and user experience for mobile devices.
then i would advice t=you have the logo anchor tag outside the `nav` tag.

- [ ] I would kindly suggest you make heading i.e Hey There I'm Austine. and `h1` tag instead of the `h3` tag you have there.
    - N.B: As stated in the requirement, The main title (e.g., "I'm Marianne ...") is a `h1` tag. If you want that google (and other search engines) knows about a specific topic, in this case, you, use an `h1` tag. [Check this link](https://www.reliablesoft.net/h1-tag/) for more info.

- [ ] Also the part where you have "I am a Software Developer" wrapped inside an `h4` tag, I would kindly suggest you make it a `p` tag. 
     - N.B: The paragraph (e.g., "I'm a software ...") is a `p` tag.This way will be semantically correct and the screen readers and other technologies will thank you

- [ ] I would kindly advice you make use of the original font family seen in figma design (i.e "Inter"). You can simply import it from google font and use it for your project. This will show the confidence in your ability to replicate any given design

- [ ]  _Please check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @brainconnect93 ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -5,36 +5,35 @@
...
     </head>
     <body>
       <header>
-        <nav>
+          <nav>
             <div class="logo-container">
                 <a href="#">
                     <h3 id="logo">My Logo</h3>
                 </a>
             </div>
```
Comment (ABDULALI3468) [inline index.html]: As requested by the previous CR (code reviewer) please consider removing the logo out of the nav tag to make it semantically correct like that.

```
<header>
      <div class="logo-container">
        <a href="#"> My Logo </a>
      </div>
      <nav>
        <label for="check" class="hamberger-btn">
          <img id="ham" src="img/IconMenu.png" alt="" />
        </label>
      </nav>
    </header>
```

File: styles.css [hunk 3]
```diff
@@ -1,137 +1,146 @@
...
   font-weight: 400;
-  font-size: 72px;
-  line-height: 82px;
-
+  font-size: 350%;
+  line-height: 64px;
+  letter-spacing: 0.37px;
+  padding-top: 15%;
+  padding-left: 5%;
 }
```
Comment (ABDULALI3468) [inline styles.css]: Please consider using the same `font-family and color properties etc` as used in the Figma template to meet the project requirements as you need to make an exact look-alike of the design.


```
font-family: 'Crete Round';
color: #FF6B00;
```

File: styles.css [hunk 4]
```diff
@@ -1,137 +1,146 @@
...
-
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 24px;
+  line-height: 32px;
+  align-items: center;
+  padding-left: 5%;
+  padding-top: 5%;
 }
```
Comment (ABDULALI3468) [inline styles.css]: Please consider using the same `color property etc` as used in the Figma template to meet the project requirements as you need to make an exact look-alike of the design.


```
color: #FFF5E1;
```

File: styles.css [hunk 5]
```diff
@@ -1,137 +1,146 @@
...
+  width: 100%;
   color: aliceblue;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  padding-left: 5%;
+  padding-top: 5%;
 }
```
Comment (ABDULALI3468) [inline styles.css]: Please consider using the same `font-family and color properties etc` as used in the Figma template to meet the project requirements as you need to make an exact look-alike of the design.


```
font-family: 'Poppins';
color: #FFFFFF;
```

File: index.html [hunk 6]
```diff
@@ -5,36 +5,35 @@
...
+            <label for="check" class="hamberger-btn">
+             <img id="ham" href="#" src="img/IconMenu.png">
+            </label>
+          </nav>
       </header>
 
       <section class="section">
         <div class="container">
-           <div class="headline">
-            <h3 id="myName">
```
Comment (ABDULALI3468) [inline index.html:29]: Please consider adding a proper `background image` to the section as can see in the design template to make it look alike to the design.

### YOUR DESIGN:-

image


### FIGMA DESIGN:-

image

**Review by ABDULALI3468 — CHANGES_REQUESTED** [review]
# Changes Required 👍 

Hi @Obimbo07  , 



Great work on making the changes requested by a previous reviewer 👏 
You've done really well implementing some of the requested changes, but there are still some which aren't been addressed yet.


## Highlights 
✅All linters are checked.
✅Project is professionally documented.
✅The right GitHub flow method has been used.
✅Professional pull request title and description.


 ## Required Changes ♻️

> **_Check the comments below to have a look at the required changes._**

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

## Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification.

> * This is My **GitHub**: GitHub followers
> * This is My **Linkedin**: [Linkedin Badge](https://www.linkedin.com/in/abdul-ali-5400bb216/)&nbsp;
> * This is My **Twitter**: [Twitter Badge](https://twitter.com/mrabdul_ali)&nbsp;

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css (hunk 5 above)
Comment (Obimbo07) [reply styles.css]: why does my stylelint give me and error "Unexpected missing generic font family "when I only use the Figma requirements

**Review by usorfaitheloho — APPROVED** [review]
Hi @Obimbo07  👋,

# STATUS: APPROVED 🕺🥳 

Congratulations🍾🥳!! Your project is ready for presentation!  There is nothing else to say other than... it's time to merge it :shipit: 



Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**